src/lib/relay/routes.tsx

```diff
@@ -54,15 +54,20 @@ class Gene extends Relay.Route {
 
 class WorksForYou extends Relay.Route {
   static queries = {
-    me: (component, params) => Relay.QL`
+    viewer: (component, params) => Relay.QL`
       query {
-        me {
-          ${component.getFragment("me", params)}
+        viewer {
+          ${(component.getFragment("viewer", params))}
```

Comment:
I actually don't even know if that would work since you can only have one fragment in a root and I need more than just the `me` fragment.